src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/service/AdjudicationMappingService.kt

```diff
@@ -136,8 +136,8 @@ class AdjudicationMappingService(
       adjudicationPunishmentMappingRepository.findById(dpsPunishmentId)
         ?.let {
           throw DuplicateMappingException(
-            existing = dpsPunishmentId,
-            duplicate = nomisBookingId to nomisSanctionSequence,
+            existing = it,
+            duplicate = createMappingRequest,
             messageIn = "Adjudication punishment mapping with id $dpsPunishmentId already exists",
           )
```

Comment:
these should return objects that can be read as Maps not primitive types

TBH The model around here is pretty bad and needs rethink. The Sync service treats these as Maps so it can write to Appinsights but this model treats them as Any